packs/softlayer/README.md

```diff
@@ -0,0 +1,17 @@
+# Softlayer Integration Pack
+
+Pack which allows integration with [Softlayer](https://www.softlayer.com/).
+
+## Configuration
+
+* ``username`` - Softlayer username.
+* ``api_key`` - Api key.
+* ``region`` - Not used yet, should be the default region you want to work with for instances.
```

Comment:
Later on when you add support for other APIs it would make sense to split configuration into `compute` and `storage` section (similar as Libcloud pack does it).